modules/learn/pages/data/data.adoc

```diff
@@ -132,7 +132,7 @@ xref:clusters-and-availability/xdcr-conflict-resolution.adoc[XDCR Conflict Resol
 If non-zero, this determines the point at which the document is removed from the system.
 The value can be set either per-document, by means of Couchbase SDK APIs (which is referred to as _TTL_); or per bucket, by means of Couchbase Web Console, the Couchbase CLI, or the Couchbase REST API (which is referred to as _Bucket TTL_).
 +
-For information on _TTL_, see xref:java-sdk::core-operations.adoc[Core Operations].
+For information on _TTL_, see xref:java-sdk:howtos:kv-operations.adoc#expiration-ttl[Core Operations].
```

Comment:
```suggestion
For information on _TTL_, see xref:java-sdk:howtos:kv-operations.adoc[Key Value Operations].
```

I don't see the `#expiration-ttl` ID on this page. Is this ID on an included partial, or perhaps you're planning to add it to the page directly in a parallel PR? The suggestion above is a potential alternative if you don't have a direct target for this at the moment.

Reply:
Thanks - will fix with Java's version of the anchor tag...